src/app/flight-booker/flight-booker.component.html

```diff
@@ -1 +1,27 @@
-<p>flight-booker works!</p>
+
+  <alert type="info"  [style.display]="display=='show'?'':'none'">
+    <p>{{alertBox}}</p>
+   </alert>
+ 
+   <section class="container d-block w-25 p-1  bg-secondary text-white rounded">
+     <h6 class="text-center">Flight Booker</h6>
+     <div class=" container d-block justify-content-center align-items-center p-3 bg-white text-dark w-100 rounded">
```

Comment:
formGroup should have been started from this point.
Then this formGroup will have various controls - journeyType (one-way/return), startDate and EndDate.